common/djangoapps/student/migrations/0011_auto_20170214_1200.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def add_email_uniqueness_constraint(apps, schema_editor):
+    # Do we already have an email uniqueness constraint?
+    cursor = schema_editor.connection.cursor()
+    cursor.execute("""
+        select * from information_schema.table_constraints
+        where constraint_schema = 'edxapp'
```

Comment:
This isn't the name of our schema in production.  I think you need to do it dynamically.